web/docker/nginx.conf.template

```diff
@@ -0,0 +1,35 @@
+log_format json escape=json '{'
+  '"body_bytes_sent": "$body_bytes_sent",'
+  '"http_referer": "$http_referer",'
+  '"http_user_agent": "$http_user_agent",'
+  '"remote_ip": "$remote_addr",'
+  '"remote_user": "$remote_user",'
+  '"request": "$request",'
+  '"request_id": "$request_id",'
+  '"request_method": "$request_method",'
+  '"request_time": "$request_time",'
+  '"request_uri": "$request_uri",'
+  '"server_name": "$server_name",'
+  '"status": "$status",'
+  '"time": "$time_iso8601"'
+'}';
```

Comment:
I accessed the container via `curl` and it will be like this:

> {"body_bytes_sent": "739","http_referer": "","http_user_agent": "curl/8.7.1","remote_ip": "xxx.xxx.xxx.xxx","remote_user": "","request": "GET / HTTP/1.1","request_id": "fde3377155ab9765825dff57cb2a83f6","request_method": "GET","request_time": "0.000","request_uri": "/","server_name": "_","status": "200","time": "2024-10-08T22:48:41+00:00"}